### ISSUE TITLE
Change something in "installation.rst"

### ISSUE THREAD

**Issue by sakuragasaki46** [django-wiki/django-wiki#1036]
When installing "wiki", following the docs (I am new to Django), I had two issues.

First, I tried to do the migration before adding the requested templates (like the docs do), but:

```
SystemCheckError: System check identified some issues:

ERRORS:
?: (wiki.E009) needs sekizai.context_processors.sekizai in TEMPLATE['OPTIONS']['context_processors']
```

I retried after adding the template directives and everything works fine.

Then, the docs refer to importing from `django.core.urlresolvers`. This in Django 2 has changed to `django.urls`, and the old import does not work anymore.

So, I propose two changes to `installation.rst`: move the migrations after the template directives and update the reference to `django.core.urlresolvers` (by either replacing or adding the Django 2 alternative).

**Comment by benjaoming** [comment]
Hi @sakuragasaki46 

Thanks for opening the issue and explaining the problem! Both changes are very welcome, and it's a good suggestion to move the migration section down after the context processor section.

I suppose that the reference to `urlresolvers` is this one? - it definitely hasn't been updated https://django-wiki.readthedocs.io/en/master/installation.html#user-account-handling

Would you be able to open a PR with these changes?

**Comment by snoop2head** [comment]
I agree to @sakuragasaki46 's points. 
I encountered the same situation. Installation doc needs to be updated according to his/her point.

**Comment by benjaoming** [comment]
Fixed in #1051 - thanks a lot @snoop2head
